common/lib/xmodule/xmodule/course_module.py

```diff
@@ -475,6 +475,26 @@ class CourseFields(object):
         # Ensure that courses imported from XML keep their image
         default="images_course_image.jpg"
     )
+    hero_image = String(
+        display_name=_("Course Hero Image"),
+        help=_(
+            "Edit the name of the hero image file. You must upload this file on the Files & Uploads page. "
```

Comment:
It seems odd to say you must upload this on the File & Uploads page-- you'd have to know what special name to use. I'd just say that it should be set on the Settings & Details page (and get rid of hero throughout, including the name of the field).